documents/Final Report/evaluation.tex

```diff
@@ -3,20 +3,24 @@ \chapter{Evaluation and failure analysis}
 
 \section{Evaluation of the connector}
 The connector is pretty much where it needs to be for the agent.
-Some of the Percepts or custom actions might need some fine tuning to increase performance, but that is not really that important since a human player wouldn't respond immediately either.
-For example the Buildings percept is so large that updating it takes a good minute at the least, which, while not necessarily a problem, is quite annoying.
+Some of the Percepts (the information the agent receives from the environment) or custom actions might need some fine tuning to increase performance, but that is not really that important since a human player wouldn't respond immediately either.
+For example the Buildings percept contains so much data that updating it takes a good minute at the least, which, while not necessarily a problem, is quite annoying.
```

Comment:
"at the least" shoul be "at least" or "at the very least"

http://dictionary.cambridge.org/grammar/british-grammar/least-the-least-at-least